src/test/isolation2/output/parallel_retrieve_cursor/explain.source

```diff
@@ -115,48 +115,52 @@ VACUUM ANALYZE pg_class;
-   Output: oid, relname, relnamespace, reltype, reloftype, relowner, relam, relfilenode, reltablespace, relpages, reltuples, relallvisible, reltoastrelid, relhasindex, relisshared, relpersistence, relkind, relnatts, relchecks, relhasrules, relhastriggers, relhassubclass, relrowsecurity, relforcerowsecurity, relispopulated, relreplident, relispartition, relisivm, relrewrite, relfrozenxid, relminmxid, relacl, reloptions, relpartbound 
- Optimizer: Postgres query optimizer                                                                                                                                                                                                                                                                                                                                                                                                                
-(3 rows)
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ Seq Scan on pg_catalog.pg_class                                                                                                                                                                                                                                                                                                                                                                                                                                  
+   Output: oid, relname, relnamespace, reltype, reloftype, relowner, relam, relfilenode, reltablespace, relpages, reltuples, relallvisible, reltoastrelid, relhasindex, relisshared, relpersistence, relkind, relnatts, relchecks, relhasrules, relhastriggers, relhassubclass, relrowsecurity, relforcerowsecurity, relispopulated, relreplident, relispartition, relisivm, relisdynamic, relrewrite, relfrozenxid, relminmxid, relacl, reloptions, relpartbound 
+ Settings: enable_incremental_sort = 'on'                                                                                                                                                                                                                                                                                                                                                                                                                         
```

Comment:
Config `enable_incremental_sort = on` manually when run tests?  Seems we should not have this GUC changed.

Reply:
> Config `enable_incremental_sort = on` manually when run tests? Seems we should not have this GUC changed.

@avamingli From a test execution perspective, this was not intentionally changed by me. The explain.source file has this enabled by default. If this needs to be changed, it should be performed with another PR? yes/no

Reply:
Ok, yes.